src/main/webapp/projectContainer/SubmissionForm.js

```diff
@@ -80,8 +79,7 @@ class SubmissionForm extends Component {
 
   componentWillUnmount() {
     this._isMounted = false;
```

Comment:
Please review if showSpinner is needed on componentDidMount and add it required.

Reply:
It was needed, so I added our spinner wrapper.